docs/Tutorials/CCE.md

```diff
@@ -100,6 +103,32 @@ values are the complex modes in m-varies-fastest format. That is,
 "Lapse_Re(2,0)", "Lapse_Im(2,0)", "Lapse_Re(2,-1)", "Lapse_Im(2,-1)",
 "Lapse_Re(2,-2)", "Lapse_Im(2,-2)"
 ```
+
+#### Spherical harmonic nodes {#cartesian_spherical_nodes}
+
+\warning This format is not yet fully supported but is under development. If you
+need it please file an issue so we can escalate the priority.
+
+In this format the value of the functions at specially chosen collocation points
+(grid points) is read in. This allows SpECTRE to perform integrals, derivatives,
+and interpolation exactly on the input data. These grid points are
+Gauss-Legendre in $cos(\theta)$ and equally spaced in $\phi$. Below is a routine
+for computing the spherical harmonic $\theta$ and $\phi$ values. These can be
+used to compute the Cartesian locations for a given radius using the standard
+transformation. The routine supports \f$\ell\in[4, 32]\f$.
+
+\snippet Test_Spherepack.cpp spectre_cce_grid_point_locations
+
+Each `dat` file holds `1 + (l_max + 1) * (2 * l_max + 1)` columns, with the
+first one being the `time`. The columns must be in \f$\theta\f$-varies-fastest
+ordering. That is,
+\f$\mathrm{time}, (\theta_0, \phi_0), (\theta_1, \phi_0), (\theta_2, \phi_0),\f$
```

Comment:
[optional] To make this consistent with the modal section do

```
\```
"time", "Theta_0_Phi_0", "Theta_1_Phi_0", ...
\```
```

Reply:
I would say the modal section should be updated. The tex I think is easier to read than the underscores. Thoughts?

Reply:
I specifically liked the blocking of the modal section. Also I'm not sure if CCE cares about the legend labels, if they have to be something specific. I'd have to check